frontend/tests/features/home/homePage.test.ts

```diff
@@ -11,62 +11,120 @@ describe("Homepage", () => {
     expect(await page.getByTestId("logo").count()).toBe(2);
     await expect(page.getByTestId(COLLECTION_LINK)).toBeVisible();
 
-    // check page scroll
-    expect(await scrollToPageBottom(page)).toBeTruthy;
```

Comment:
this is not enough to test if a page is scrollable, because it programmatically goes to the bottom of the page, which gets around CSS caused unscrollable page. So we need to actually executes wheel scrolls

For example, when `overflow-y: hidden`, `scrollToPageBottom()` still works but the user actually can't scroll the page